app/routes/_index/dashboard.tsx

```diff
@@ -49,9 +54,9 @@ export function Dashboard() {
 				/>
 				<QuickStat
 					title="Rounds Played"
-					stat="6"
-					trend="2 fewer than this time last month"
-					trendDirection="negative"
+					stat={`${rounds.length}`}
```

Comment:
we can make the `stat` prop `string | number` - number seems more appropriate anyways 😅

Reply:
Updated to be `string | number`